test/frontend/cast.jl

```diff
@@ -91,7 +92,7 @@ function foo(
     @test cmd.options["option1"].name == "option1"
     @test cmd.options["option2"].name == "option2"
     @test cmd.options["o"].short == true
-    @test cmd.options["o"].hint === nothing
+    @test cmd.options["o"].hint === "nothing"
```

Comment:
i'm not 100% sure about this. hint is suppose to be a `String` so I think the previous test was wrong?

Reply:
yes, previously the hint will be missing so it wasn't string but nothing... but I think now this is the right fix